packages/run-utils/package.json

```diff
@@ -21,10 +48,6 @@
     "ava": "^2.4.0",
     "semver": "^7.3.2"
   },
-  "repository": {
```

Comment:
Should we keep this format and specify a directory for the sub packages as described here:
https://docs.npmjs.com/files/package.json#repository

Reply:
Good catch! I just fixed it.